apps/web/src/dialogs/settings/backup-export-settings.ts

```diff
@@ -164,7 +167,10 @@ export const BackupExportSettings: SettingsGroup[] = [
       {
         key: "backup-directory",
         title: strings.selectBackupDir(),
-        description: strings.selectBackupDirDesc(),
+        description: () =>
+          strings
+            .selectBackupDirDesc(getDesktopBackupsDirectoryPath())
```

Comment:
This won't update when user changes their backup directory path.

Reply:
It is updating for me. When I change the backup directory path, the path text shown in the description also changes. Is there something else you're referring to here?

Reply:
Let me see.

Reply:
It's working.